cloud/blockstore/libs/nbd/netlink_device.cpp

```diff
@@ -0,0 +1,454 @@
+#include "client_handler.h"
+#include "netlink.h"
+#include "netlink_device.h"
+#include "utils.h"
+
+#include <cloud/storage/core/libs/diagnostics/logging.h>
+#include <netlink/genl/ctrl.h>
+#include <netlink/genl/genl.h>
+#include <netlink/netlink.h>
+#include <util/generic/scope.h>
+
+namespace NCloud::NBlockStore::NBD {
+
+namespace {
+
+#undef NLA_PUT
```

Comment:
Instead of redefining `NLA_PUT`, consider renaming this macro to avoid conflicts with the netlink headers.

Reply:
Name is not the reason I'm redefining it. All NLA_PUT_U* macro use it and original version expects a label named nla_put_failure to goto if error occured